### PR TITLE
Fix README deployment steps

### PR DESCRIPTION
**Pull request by BorjaEst**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,18 +32,19 @@ USERNAME
 PASSWORD
 ```
 
-7. Run `docker-compose build`
+7. Run `sudo docker-compose  -f docker-compose.yaml -f docker-compose.prod.yaml build`
```
Comment (TheChristophe) [inline README.md:35]: sudo is not required on a properly configured system

File: README.md [hunk 2]
```diff
@@ -32,18 +32,19 @@ USERNAME
...
 8. Run backend build steps in a venv
-9. Run `docker-compose up`
+9. Run `sudo docker-compose  -f docker-compose.yaml -f docker-compose.prod.yaml up`
 
 #### To reset your database
 
-Run `./scripts/reset-database.sh` (Linux) or `./scripts/reset-database.ps1` (Windows)
+1. Run `cd scripts`
```
Comment (TheChristophe) [inline README.md:41]: Running from project folder was already correct: the scripts will run `docker-compose` in the current / root folder, which is also where docker-compose.yaml is. I do not know the semantics of how docker-compose behaves if you run it in a child folder.

Comment (BorjaEst) [reply README.md:41]: Ok, I roll back this one.
But, in that case, would not be clear that all the script `sandbox.py` in "scripts" would run correctly also from the root folder?

I think that was the reason of our confusion.

File: README.md (hunk 1 above)
Comment (BorjaEst) [reply README.md:35]: ok

File: README.md [hunk 3]
```diff
@@ -38,8 +38,7 @@ PASSWORD
 
 #### To reset your database
 
-1. Run `cd scripts` 
-2. Run `./reset-database.sh` (Linux) or `./reset-database.ps1` (Windows)
+1. Run `./reset-database.sh` (Linux) or `./reset-database.ps1` (Windows)
```
Comment (TheChristophe) [inline README.md:41]: Needs to be ./scripts/reset-database

**Review by TheChristophe — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (TheChristophe) [reply README.md]: sandbox.py would, I believe, not run outside the script folder, as it opens files relative to current folder

**Pull request merged**